apps/studio/src/features/mail/templates/templates.ts

```diff
@@ -50,11 +54,47 @@ export const invitationTemplate = (
   }
 }
 
+export const publishAlertContentPublisherTemplate = (
+  data: PublishAlertContentPublisherEmailTemplateData,
+): EmailTemplate => {
+  const { recipientEmail, siteName, resource } = data
+  const resourceStudioUrl = getResourceStudioUrl(resource)
```

Comment:
[nit] `studioResourceUrl`

Reply:
updated in https://github.com/opengovsg/isomer/pull/1349/commits/9faba3d4858aff37c312dbc18501fc9bf8821e57 !